Public/Src/Cache/ContentStore/Library/Service/Grpc/GrpcCopyClient.cs

```diff
@@ -135,15 +172,8 @@ private async Task<T> RunClientActionAndThrowIfFailedAsync<T>(Context context, F
         /// <inheritdoc />
         public void Dispose()
         {
-            Dispose(false);
-            GC.SuppressFinalize(this);
+            // noop for now

```

Comment:
Shouldn't at least the `_clientPool` be emptied?

Reply:
Removing _clientPool

---
In reply to: [271397083](https://github.com/Microsoft/BuildXL/pull/66#discussion_r271397083) [](ancestors = 271397083)